.github/workflows/tests.yml

```diff
@@ -86,13 +86,4 @@ jobs:
         run: "composer run-script test:unit"
 
       - name: "Run Larastan on a sample Laravel application"
-        run: "tests/laravel-test.sh ^9.0"
-        if: "matrix.laravel == '^9.52.16'"
-
-      - name: "Run Larastan on a sample Laravel application"
-        run: "tests/laravel-test.sh ^10.0"
-        if: "matrix.laravel == '^10.28.0'"
-
-#      - name: "Run Larastan on a sample Laravel application"
-#        run: "tests/laravel-test.sh dev-master"
-#        if: "matrix.laravel == '^11.0'"
+        run: "version=${{ matrix.laravel }}; tests/laravel-test.sh ${version%%.*}.0"
```

Comment:
```suggestion
        run: |
          version="${{ matrix.laravel }}"
          tests/laravel-test.sh "${version%%.*}.0"
```

Reply:
That works, I was trying to keep it a one-liner